dali/pipeline/pipeline.h

```diff
@@ -310,6 +309,15 @@ class DLL_PUBLIC Pipeline {
     async_execution_ = async_execution;
   }
 
+  /**
+   * @brief Set if the DALI pipeline should operator output buffer statistics
+   *
+   * @param get_memory_stats If DALI should print operator output buffer statistics.
+   * Usefull for `bytes_per_sample_hint` operator parameter.
+   */
+  DLL_PUBLIC void DumpOperatorOutputMemoryStatistics(bool get_memory_stats = true) {
+    get_memory_stats_ = get_memory_stats;
+  }
```

Comment:
That deserves a section in Advanced guide how to set the hint in valid way for the types of used operators based on output (I don't remember the exact code, but I think CPU divides the hint for all the Tensors in Batch).

Reply:
Done